docs/usage/prerequiste.md

```diff
@@ -0,0 +1,85 @@
+# Setting up the usage environment
+
+:warning: All paths are relative to the root location of this project repo
+
+- Run the Node Controller Manager either as described in [Setting up a local development environment](development/local_setup.md) or [Deploying the Node Controller Manager into a Kubernetes cluster](deployment/kubernetes.md).
+- Make sure that you have run the following steps before managing instances/ instance-sets/ instance-deploys.
+
+## Make KUBECONFIG point to your cluster
+
+Now using the existing [Kubeconfig](https://kubernetes.io/docs/tasks/access-application-cluster/authenticate-across-clusters-kubeconfig/), please open another Terminal panel/window with the `KUBECONFIG` environment variable pointing to this Kubeconfig file as shown below,
+
+```bash
+$ export KUBECONFIG=$GOPATH/src/github.com/gardener/node-controller-manager/dev/kubeconfig.yaml
+```
+## Replace provider credentials and desired VM configurations
+
+Now open `kubernetes/aws-instance-class.yaml` and replace required values there with your desired VMs configuration. 
+
+Similarily open `kubernetes/secret.yaml` and replace - *userData, providerAccessKeyId, providerSecretAccessKey* with base64 encoded values of cloudconfig file, AWS access key id, and AWS secret access key respectively. You can use the following command to get the base64 encoded value of your details
+
+```bash
+$ echo "sample-cloud-config" | base64
+base64-encoded-cloud-config
+```
+
+Do the same for your access key and secret key.
+
+## Deploy required CRDs and Objects
+
+Create all the required CRDs in the cluster using `kubernetes/crds.yaml`
+```bash
+$ kubectl apply -f kubernetes/crds.yaml
+```
+
+Create the class template that would be used as an instance template to create your VMs using `kubernetes/aws-instance-class.yaml`
```

Comment:
s/that would be used as an instance template to create your VMs/that will be used as an instance template to create VMs/